source/agora/consensus/Fee.d

```diff
@@ -427,17 +456,19 @@ public class FeeManager
             foreach (input; tx.inputs)
             {
                 UTXO utxo;
-                assert(peekUTXO(input.utxo, utxo));
+                if (!peekUTXO(input.utxo, utxo))
+                    return "unable to find input for utxo";
                 tot_in.mustAdd(utxo.output.value);
             }
 
-            assert(tx.getSumOutput(tot_out), "Not validated block in" ~
-                "getTXSetFees");
+            if (!tx.getSumOutput(tot_out))
+                return "Not validated block in getTXSetFees";
```

Comment:
That message doesn't make much sense, think we can reword it ?

Reply:
I can rename it, it came from a previous commit though